lib/bot/BotCommands.js

```diff
@@ -168,31 +152,20 @@ const BotCommands = {
     return fromUser + ' :8ball: ' + reply + ' :sparkles:';
   },
 
-  camperCount: function() {
-    return 'WIP camperCount';
-  },
-
-  welcome: function(input) {
-    if (input.params && input.params.match(/world/i)) {
-      return '## welcome to FreeCodeCamp @' +
-        input.message.model.fromUser.username + '!';
-    }
-    return null;
-  },
-
-  hello: function(input, bot) {
-    return (this.welcome(input, bot) );
+  wiki: function() {
+    return '#### Our wiki : ' +
+    '\n:point_right: Our wiki has moved to ' +
```

Comment:
How long should we acknowledge that the wiki has a new home? Perhaps change `Our wiki has move to` to `The freeCodeCamp wiki can be found on`